File: Ruby/rankB_Up/SortingCharacterAndIntegerPairs/step2.rb
# 1行目に行数を表す整数 n、続く n 行で m 個の「文字」と「整数」の組が空白区切りで入力されます。
# n 個の整数だけをそのまま順に出力してください。

n = gets.to_i

n.times do
    s, d = gets.chomp.split(" ")
    puts d
end

File: Python/rankB_Up/SortingCharacterAndIntegerPairs/step2.py
# 1行目で整数 n が与えられ、2行目で n 個の整数が与えられます。
# n 個の整数を昇順に出力してください。

n = int(input())
for _ in range(n):
    s, d = input().split()
    print(d)